security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolationDetector.java

```diff
@@ -36,6 +36,19 @@ default void checkCurrent(Contingency contingency, Branch branch, Branch.Side si
         checkCurrent(branch, side, currentValue, consumer);
     }
 
+    /**
+     * Mirror checkCurrent on {@link Branch} but for {@link ThreeWindingsTransformer} instead.
```

Comment:
This is not a very convenient Javadoc comment. The user will need find the "good" referenced method in order to know what the current method do.

You should replace **all** the javadoc comments beginning with "Mirror...", by an adapted copy of the referenced method comment. This way, each method will have a clear documentation.

Note that when overriding or implementing a method, if no javadoc comment is provided, the one of the overridden method is automatically used.

Reply:
done